src/main/resources/config/liquibase/changelog/20190715204745_changelog.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+
+    <property name="autoIncrement" value="true"/>
+    <!--
+        Added the entity ExerciseHint.
+    -->
+    <changeSet id="20190715204744-1" author="jhipster">
+        <createTable tableName="exercise_hint">
+            <column name="id" type="bigint" autoIncrement="${autoIncrement}">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="title" type="varchar(255)">
+                <constraints nullable="true" />
+            </column>
+
+            <column name="content" type="varchar(2000)">
```

Comment:
I am wondering if 2000 is enough. Should we directly use `LONGTEXT` here?

Reply:
I think you are absolutely right!